README.md

```diff
@@ -1,6 +1,8 @@
 IBAMR
 =====
 
+:robot:
+
```

Comment:
Are we going to have `build : passing` here soon?

Reply:
If you want a badge in the readme, [that is a separate thing](https://github.com/brennonbrimhall/CBadge). Injecting javascript into the comments here would likely be difficult due to security policies.